--- node_modules/@cosmjs/cosmwasm-stargate/build/signingcosmwasmclient.js	2021-08-26 14:05:40.290936121 +0300
+++ signingcosmwasmclient.js	2021-08-26 14:08:50.623151369 +0300
@@ -76,7 +76,7 @@
         return new SigningCosmWasmClient(undefined, signer, options);
     }
     /** Uploads code and returns a receipt, including the code ID */
-    async upload(senderAddress, wasmCode, meta = {}, memo = "") {
+    async upload(senderAddress, wasmCode, fee, meta = {}, memo = "") {
         const source = meta.source || "";
         const builder = prepareBuilder(meta.builder);
         const compressed = pako_1.default.gzip(wasmCode, { level: 9 });
@@ -89,7 +89,7 @@
                 builder: builder,
             }),
         };
-        const result = await this.signAndBroadcast(senderAddress, [storeCodeMsg], this.fees.upload, memo);
+        const result = await this.signAndBroadcast(senderAddress, [storeCodeMsg], fee, memo);
         if (stargate_1.isBroadcastTxFailure(result)) {
             throw new Error(createBroadcastTxErrorMessage(result));
         }
@@ -105,7 +105,7 @@
             transactionHash: result.transactionHash,
         };
     }
-    async instantiate(senderAddress, codeId, msg, label, options = {}) {
+    async instantiate(senderAddress, codeId, msg, label, fee, options = {}) {
         const instantiateContractMsg = {
             typeUrl: "/cosmwasm.wasm.v1beta1.MsgInstantiateContract",
             value: tx_4.MsgInstantiateContract.fromPartial({
@@ -117,7 +117,7 @@
                 admin: options.admin,
             }),
         };
-        const result = await this.signAndBroadcast(senderAddress, [instantiateContractMsg], this.fees.init, options.memo);
+        const result = await this.signAndBroadcast(senderAddress, [instantiateContractMsg], fee, options.memo);
         if (stargate_1.isBroadcastTxFailure(result)) {
             throw new Error(createBroadcastTxErrorMessage(result));
         }
